server/routes.js

```diff
@@ -188,7 +188,19 @@ router.post('/api/domains/:domain/workflows/:workflowId/:runId/signal/:signal',
 
 router.get('/api/domains/:domain/workflows/:workflowId/:runId', async function (ctx) {
   try {
-    ctx.body = await ctx.cadence.describeWorkflow();
+    const describeResponse = await ctx.cadence.describeWorkflow();
+
+    if (describeResponse.workflowExecutionInfo) {
+      describeResponse.workflowExecutionInfo.closeEvent = null;
```

Comment:
Why is `describeResponse.workflowExecutionInfo.closeEvent = null;` necessary? Shouldn't it be ok that it's undefined?

Reply:
Should be fine as undefined. Just making sure to define it in the response from an API to show what to expect from a contract perspective to the frontend.